packages/runtime/src/types.ts

```diff
@@ -270,62 +280,62 @@ export interface DeploySSCParam extends BasicParams, SSCOptionalFlags {
   globalBytes: number
   approvalProg?: Uint8Array
   clearProg?: Uint8Array
-}
+};
 
-export interface OptInSSCParam extends BasicParams, SSCOptionalFlags {
+export type OptInSSCParam = BasicParams & SSCOptionalFlags & {
```

Comment:
what is the difference between interface and types ? why we are using type here ?

Reply:
With interface i wanted to do something like: 
```js
interface BasicParams extends (SignWithLsig | SignWithSk) { .. }
```
But it's not possible with interfaces (extending unions). And this is the correct way because *lsig* should not be optional, it should be a required param if SignType is logic signature. Also we don't need to take complete account in this case, only [`from`, `to`] addresses. 

Read more about the difference here - [https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c](https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c). Picked `type` because of last points in this article.

Reply:
With type you create a type variable (like `var`, `const` ....) - so you need assignment. 
With interface you create a named type.